avm/res/api-management/service/tests/e2e/defaults.v2Sku/main.test.bicep

```diff
@@ -46,6 +46,14 @@ module testDeployment '../../../main.bicep' = [
       sku: 'BasicV2'
       publisherEmail: 'apimgmt-noreply@mail.windowsazure.com'
       publisherName: '${namePrefix}-az-amorg-x-001'
+      managedIdentities: {
+        systemAssigned: true
+      }
+      tags: {
```

Comment:
We can avoid using tags for cleaner code if that does not add value to the specific sku test. This comment applied to all other tests as well